CMakeLists.txt

```diff
@@ -41,6 +41,25 @@ if(APPLE)
   endif()
 endif()
 
+# Enable 'FORCE_ANSI_COLORED_OUTPUT' option when using Ninja.
+# Forces GCC/Clang compilers to enable color diagnostics. Enabled by default.
+if(CMAKE_GENERATOR STREQUAL "Ninja")
```

Comment:
Does Make support this as well?

Reply:
Make works differently (not sure exactly about the details) but it doesn't need to force color diagnostics.
but yeah we could enable it for both generator options, although not necessary